### PR TITLE
Issue 672 sidebar menu adjustments

### PR DESCRIPTION
**Pull request by adaudenthun**
#672 

Change icons to button and adjust the style

### CODE REVIEW

File: src/themes/default/components/core/blocks/SidebarMenu/SidebarMenu.vue [hunk 1]
```diff
@@ -35,7 +35,7 @@
             >
               {{ category.name }}
             </router-link>
-            <sub-btn class="flex-end center-self" :id="category.id"/>
+            <sub-btn class="flex-end w-50 center-self bg-transparent brdr-none align-right" :id="category.id"/>
```
Comment (Aekal) [inline src/themes/default/components/core/blocks/SidebarMenu/SidebarMenu.vue]: `flex-end` and `center-self` classes are unnecessary here

File: src/themes/default/components/core/blocks/SidebarMenu/SidebarMenu.vue [hunk 2]
```diff
@@ -53,11 +53,11 @@
               {{ $t('Track my order') }}
             </router-link>
           </li>
-          <li @click="closeMenu" class="brdr-bottom brdr-c-alto flex">
+          <li @click="closeMenu" >
             <router-link v-if="currentUser" class="block px25 py20 c-darkgray no-underline col-xs" to="/my-account" exact>
               {{ $t('My account') }}
             </router-link>
-            <sub-btn v-if="currentUser" class="flex-end center-self"/>
+            <sub-btn v-if="currentUser" class="flex-end w-50 center-self bg-transparent brdr-none align-right"/>
```
Comment (Aekal) [inline src/themes/default/components/core/blocks/SidebarMenu/SidebarMenu.vue]: Remove `flex-end` class

**Comment by pkarw** [comment]
@adaudenthun we need this PR to be merged with 1.0RC (before 1st of March). Could you please make the changes in the next 2 days?

**Comment by adaudenthun** [comment]
Yes sorry , i do this tonight.

**Comment by adaudenthun** [comment]
I push the modification ;), you can show it and test it.

**Comment by adaudenthun** [comment]
Hi i made the changes, you can test now, thanks.

**Comment by pkarw** [comment]
Thanks! @Aekal can you confirm this and merge in?

**Comment by Aekal** [comment]
@adaudenthun It's much better now, thanks! But I still see some issues

These icons should be hidden on mobile
zrzut ekranu 2018-02-26 o 21 19 30
After login arrow icon doesn't fit in a line
zrzut ekranu 2018-02-26 o 21 24 52

**Comment by pkarw** [comment]
@adaudenthun can we somehow help you with finishing this task? It would be great to finalize it and merge in :)

**Comment by filrak** [comment]
@aekal maybe you want to finish this task?

**Comment by Aekal** [comment]
@filrak Ok I'll finish it next week, I have to go offline for few days ;)

**Comment by filrak** [comment]
@Aekal sure, thank you!

**Pull request merged**